<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParForm</class>
 <widget class="QWidget" name="ParForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>210</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Resposta Correta:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbBitsDados">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>lbBitsDados</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>bit de paridade par</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>60</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string>Legenda:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>189</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Bit Errado</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbRespostaCorreta">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>lbRespostaCorreta</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>-10</y>
     <width>189</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Bit de Paridade</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>189</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Bit de Dados</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bits Informados:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>189</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>bit de paridade Ã­mpar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(0, 0, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbNum">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:blue;
</string>
   </property>
   <property name="text">
    <string>lnNum</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbNum_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:blue;</string>
   </property>
   <property name="text">
    <string>lnNum</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:blue;
</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:blue;
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tf_Informacoes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>501</width>
     <height>70</height>
    </rect>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
